config/jobs/image-pushing/k8s-staging-kind.yaml

```diff
@@ -25,3 +25,81 @@ postsubmits:
       rerun_auth_config:
         github_team_ids:
           - 2921947 # kind-maintainers
+    - name: post-kind-push-base-image
+      cluster: k8s-infra-prow-build-trusted
+      run_if_changed: '^images\/base'
+      annotations:
+        testgrid-dashboards: sig-testing-kind, wg-k8s-infra-gcb
+        testgrid-alert-email: kubernetes-engprod+alerts@google.com,antonio.ojea.garcia@gmail.com
```

Comment:
we should stop sending it to this alias, kubernetes-engprod isn't staffing kind. we can just put bentheelder@google.com